trunk/SUAVE/Methods/Performance/V_n_diagram.py

```diff
@@ -0,0 +1,990 @@
+
+    Inputs:
+    V_n_data.
+        airspeeds.positive                      [kts]
+            negative                            [kts]
+        weight                                  [lb]
+        density                                 [slug/ft**3]
+        density_ratio                           [Unitless]
+        reference_area                          [ft**2]
+        maximum_lift_coefficient                [Unitless]
+        minimum_lift_coefficient                [Unitless]
+        load_factors.positive                   [Unitless]
+        load_factors.negative                   [Unitless]
+
+    Outputs:
+    V_n_data.
+        airspeeds.positive                      [kts]
+            negative                            [kts]
+        Vs1.positive                            [kts]       
+            negative                            [kts]
+        Va.positive                             [kts]
+            negative                            [kts]
+        load_factors.positive                   [Unitless]
+            negative                            [Unitless]                 
+
+    Properties Used:
+    N/A
+
+    Description:   
+    """     
+
+    # Unpack
+    airspeeds_pos    = V_n_data.airspeeds.positive
+    airspeeds_neg    = V_n_data.airspeeds.negative
+    weight           = V_n_data.weight
+    rho              = V_n_data.density
+    reference_area   = V_n_data.reference_area
+    max_lift_coef    = V_n_data.maximum_lift_coefficient
+    min_lift_coef    = V_n_data.minimum_lift_coefficient
+    load_factors_pos = V_n_data.load_factors.positive
+    load_factors_neg = V_n_data.load_factors.negative
+    
+    # Stall speeds
+    airspeeds_pos[1] = (2 * weight / (rho * reference_area * max_lift_coef)) ** 0.5
+    airspeeds_neg[1] = (2 * weight / (rho * reference_area * abs(min_lift_coef))) ** 0.5
+    airspeeds_pos[0] = airspeeds_pos[1]
+    airspeeds_neg[0] = airspeeds_neg[1]
+
+    # Maneuver speeds
+    airspeeds_pos[2] = airspeeds_pos[1] * load_factors_pos[2] ** 0.5
+    airspeeds_neg[2] = (2 * weight * abs(load_factors_neg[2]) / (rho * reference_area * \
+                                                                 abs(min_lift_coef))) ** 0.5
+    
+    # Pack
+    V_n_data.airspeeds.positive           = airspeeds_pos
+    V_n_data.airspeeds.negative           = airspeeds_neg
+    V_n_data.load_factors.positive        = load_factors_pos
+    V_n_data.load_factors.negative        = load_factors_neg
+    V_n_data.Vs1.positive                 = airspeeds_pos[1]
+    V_n_data.Vs1.negative                 = airspeeds_neg[1]
+    V_n_data.Va.positive                  = airspeeds_pos[2]
+    V_n_data.Va.negative                  = airspeeds_neg[2]
+    
+#------------------------------------------------------------------------------------------------------------
+
+def stall_line(V_n_data, upper_bound, lower_bound, Num_of_points, sign_flag):
```

Comment:
As @planes already suggested since you are passing lots of parameters, perhaps consider passing a data structure.

Reply:
Could you clarify what the maximum number of parameters I can have? For this function, it seems inconvenient to create a data structure.

Reply:
This is ok. The issue is readability when the function inputs stretch over one line.